.github/workflows/publish.yaml

```diff
@@ -0,0 +1,25 @@
+name: publish
+on:
+  push:
+    branches:    
+      - 'main'
+
+jobs:
+  build_and_publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - run: |
+          docker build . --tag ghcr.io/badgerloop-software/chase-car-dashboard-image:latest
```

Comment:
We should also tag revisions so that if we need to revert to a specific version we can just pull :<hash>

Reply:
How do you suggest we do this? We can either tag each new package creation (on push) with a new version number, or we can create an action that allows us to tag specific ones with a version number.

Reply:
You solution of getting the first 8 octets of the git hash is good (and industry standard, nice!)

Reply:
See [Issue 352](https://badgerloop.atlassian.net/browse/SW-352?atlOrigin=eyJpIjoiOTE3ZTUxZjk2MzhhNGE2YmFkYjNmNmE1YThkZmYxYTgiLCJwIjoiaiJ9) for details about how we want to tag the image